src/components/ComputeModal.js

```diff
@@ -1068,7 +1109,8 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
                   'If you want to permanently save some files from the disk before deleting it, you will need to create a new cloud environment to access it.'
                 ])
               ]),
-              existingRuntime.tool === 'RStudio' ? h(SaveFilesHelpRStudio) : h(SaveFilesHelp)
+              // At this point there is no runtime to check the tool
+              h(SaveFilesHelpRStudio)
```

Comment:
Could you explain the reason for this change a bit more? I'm trying to understand why there is no runtime at this point.